lib/travis/build/script/r.rb

```diff
@@ -25,6 +25,7 @@ class R < Script
           # Heavy dependencies
           pandoc: true,
           latex: true,
+          fortran: false,
```

Comment:
If we have an option for this, it definitely has to default to `true`. It also should work for non MacOS builds as well (just excluding `gfortran` from https://github.com/travis-ci/travis-build/pull/1193/files#diff-5b7e1aa01f1c6f058df6e0b9e2486c62R93 would do it I think)